modules/core/src/main/java/org/apache/ignite/internal/management/AbstractCommandInvoker.java

```diff
@@ -102,11 +103,20 @@ protected <A extends IgniteDataTransferObject> A argument(
     ) throws InstantiationException, IllegalAccessException {
         A arg = argCls.newInstance();
 
-        AtomicBoolean oneOfSet = new AtomicBoolean(false);
+        AtomicBoolean onlyOneOfGrp = new AtomicBoolean();
```

Comment:
May be it is better to create anonymous class that implements `Consumer` and has fields and set them approprieately?
Atomics here look quite weird, tbh

Reply:
Yes. Implemented